packages/node/src/evm/handlers/initialize-provider.ts

```diff
@@ -51,32 +50,10 @@ export async function initializeProvider(
   const state1 = state.refresh(initialState);
 
   // =================================================================
-  // STEP 2: Get current block number, and verify or set Airnode parameters
+  // STEP 2: Get current block number
```

Comment:
Nice one fixing the comment. It's easy to forget

Reply:
These comments have been extremely helpful to me when trying to follow the coordinator-worker flow and understand what's actually happening. Well done!